about.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Material+Icons+Outlined">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/mobile.css">
+    <link rel="stylesheet" href="aboutCSS/about.css">
+    <link rel="stylesheet" media="(min-width:768px)" href="aboutCSS/aboutDesktop.css">
+    <link rel="stylesheet" media="(min-width:768px)" href="css/desktop.css">
+    <title>Capstone1-About</title>
+</head>
+<body>
+    <header>
+        <div class="top">
+            <div class="media">
+                <ul class="mediaList">
+                    <li>
+                        <div class="mediaIcon">
+                            <a class="face" href="#"><img src="images/png/facebook.png" alt="Facebook icon"></a>
+                        </div>
+                    </li>
+                    <li>
+                        <div class="mediaIcon twit">
+                            <a href="#"><img src="images/png/twitter.png" alt="Twitter icon"></a>
+                        </div>
+                    </li>
+                    <li><a href="#">English</a></li>
+                    <li><a href="#">My Page</a></li>
+                    <li><a href="#">Logout</a></li>
+                </ul>
+            </div>
+            <div class="desktopBar">
+                <div class="logo">
+                    <a href="index.html"><img src="images/png/logo.png" alt="Rock logo icon"></a>
+                </div>
+                <ul class="deskOptions">
+                    <li>
+                        <a href="#" class="deskLi">About</a>
+                    </li>
```

Comment:
- [ ] It would be nice if you can change this text from `About` to `Home`. As you are already on the about page, you want a link to go back to the homepage (not only the logo). Also, don't forget to change the link.